### PR TITLE
[Docs] Create homepage for Sources/Destinations

### PR DESCRIPTION
**Pull request by nataliekwong**
These pages aren't helpful as they only list the available sources:
https://docs.airbyte.com/category/destinations
https://docs.airbyte.com/category/sources

https://docs.airbyte.com/integrations/ is more helpful because it contains a list and metadata.

Create a similar README with all the related metadata information.

### CODE REVIEW

File: docs/integrations/destinations/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+A destination is a data warehouse, data lake, database, or an analytics tool where you want to load your ingested data.
+
+Learn about our [Connector Support Levels](/integrations/connector-support-levels.md) to understand what to expect from a connector.
+
+
+## Destinations
+
+<ConnectorRegistry type="destination"/>
+
+_[View the connector registry in full](https://connectors.airbyte.com/files/generated_reports/connector_registry_report.html)_
```
Comment (evantahler) [inline docs/integrations/destinations/README.md:14]:
```suggestion
_[View the connector registry in full](/integrations)_
```

I think we'd prefer to point folks to the view here https://docs.airbyte.com/integrations.  The list on connectors.airbyte.com is probably too information dense (and doesn't scroll right).

File: docs/integrations/sources/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+A source is an API, file, database, or data warehouse that you want to ingest data from.
+
+Learn about our [Connector Support Levels](/integrations/connector-support-levels.md) to understand what to expect from a connector.
+
+## Sources
+
+<ConnectorRegistry type="source"/>
+
+_[View the connector registry in full](https://connectors.airbyte.com/files/generated_reports/connector_registry_report.html)_
```
Comment (evantahler) [inline docs/integrations/sources/README.md:13]:
```suggestion
_[View the connector registry in full](/integrations)_
```

File: docs/integrations/destinations/README.md (hunk 1 above)
Comment (nataliekwong) [reply docs/integrations/destinations/README.md]: Makes sense!

File: docs/integrations/destinations/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import ConnectorRegistry from '@site/src/components/ConnectorRegistry';
+
+# Destinations
+
+A destination is a data warehouse, data lake, database, or an analytics tool where you want to load your ingested data.
+
+Learn about our [Connector Support Levels](/integrations/connector-support-levels.md) to understand what to expect from a connector.
```
Comment (evantahler) [inline docs/integrations/destinations/README.md:7]:
```suggestion
Learn about our [Connector Support Levels](/integrations/connector-support-levels) to understand what to expect from a connector.
```

File: docs/integrations/sources/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import ConnectorRegistry from '@site/src/components/ConnectorRegistry';
+
+# Sources
+
+A source is an API, file, database, or data warehouse that you want to ingest data from.
+
+Learn about our [Connector Support Levels](/integrations/connector-support-levels.md) to understand what to expect from a connector.
```
Comment (evantahler) [inline docs/integrations/sources/README.md:7]:
```suggestion
Learn about our [Connector Support Levels](/integrations/connector-support-levels) to understand what to expect from a connector.
```

**Review by evantahler — CHANGES_REQUESTED** [review]

File: docs/integrations/destinations/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import ConnectorRegistry from '@site/src/components/ConnectorRegistry';
+
+# Destinations
+
+A destination is a data warehouse, data lake, database, or an analytics tool where you want to load your ingested data.
+
+Learn about our [Connector Support Levels](/integrations/connector-support-levels) to understand what to expect from a connector.
```
Comment (katmarkham) [inline docs/integrations/destinations/README.md:7]: learn >read more

File: docs/integrations/sources/README.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import ConnectorRegistry from '@site/src/components/ConnectorRegistry';
+
+# Sources
+
+A source is an API, file, database, or data warehouse that you want to ingest data from.
+
+Learn about our [Connector Support Levels](/integrations/connector-support-levels) to understand what to expect from a connector.
```
Comment (katmarkham) [inline docs/integrations/sources/README.md:7]: learn >read more

**Review by katmarkham — DISMISSED** [review]

File: docs/connector-development/connector-builder-ui/overview.md [hunk 7]
```diff
@@ -13,7 +13,7 @@ Developer updates will be announced via our #help-connector-development Slack ch
 
 The connector builder is the right tool if the following points are met:
 * You want to integrate with a JSON-based HTTP API as a source of records
-* The API you want to integrate with doesn't exist yet as a connector in the [connector catalog](/category/sources).
+* The API you want to integrate with doesn't exist yet as a connector in the [connector catalog](/integrations/sources/README.md).
```
Comment (evantahler) [inline docs/connector-development/connector-builder-ui/overview.md:16]:
```suggestion
* The API you want to integrate with doesn't exist yet as a connector in the [connector catalog](/integrations/sources/).
```

File: docs/integrations/destinations/README.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import ConnectorRegistry from '@site/src/components/ConnectorRegistry';
+
+# Destinations
+
+A destination is a data warehouse, data lake, database, or an analytics tool where you want to load your ingested data.
+
+Read more about our [Connector Support Levels](/integrations/connector-support-levels.md) to understand what to expect from a connector.
```
Comment (evantahler) [inline docs/integrations/destinations/README.md:7]:
```suggestion
Read more about our [Connector Support Levels](/integrations/connector-support-levels) to understand what to expect from a connector.
```

File: docs/integrations/sources/README.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+import ConnectorRegistry from '@site/src/components/ConnectorRegistry';
+
+# Sources
+
+A source is an API, file, database, or data warehouse that you want to ingest data from.
+
+Read more about our [Connector Support Levels](/integrations/connector-support-levels.md) to understand what to expect from a connector.
```
Comment (evantahler) [inline docs/integrations/sources/README.md:7]:
```suggestion
Read more about our [Connector Support Levels](/integrations/connector-support-levels) to understand what to expect from a connector.
```

**Review by evantahler — CHANGES_REQUESTED** [review]

**Review by evantahler — APPROVED** [review]

**Pull request merged**